chat-plugins/wssb.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+let fs = require('fs');
+let monData;
+try {
+	monData = fs.readFileSync("data/WSSB.txt").toString().split("\n\n");
+} catch (e) {
+	console.error(e);
+}
+
+function getMonData(target) {
+	let returnData = null;
+	monData.forEach(function (data) {
+		if (toId(data.split("\n")[0].split(" - ")[0] || " ") === target) {
+			returnData = data.split("\n").map(function (line) {
+				return Tools.escapeHTML(line);
+			}).join("<br />");
+		}
+	});
+	return returnData;
+}
+
+exports.commands = {
+	ssb: 'wssb',
+	wssb: function (target, room, user) {
+		if (!this.runBroadcast()) return false;
+		if (!target || target === 'help') return this.parse('/help Wssb');
+		if (target === 'credits') return this.parse('/wssbcredits');
+		let targetData = getMonData(toId(target));
+		if (!targetData) return this.errorReply("The staffmon '" + toId(target) + "' could not be found.");
+		return this.sendReplyBox(targetData);
+	},
+
+	ssbhelp: 'wssbhelp',
+	wssbhelp: function (target, room, user) {
+		if (!this.runBroadcast()) return;
+		return this.sendReplyBox("/Wssb [staff member\'s name] - displays data for a staffmon\'s movepool, custom move, and custom ability.");
```

Comment:
this isn't how this help command should be done... just use the standard help command system

``wssbhelp: ["Usage: /wssb [staff member's name] - displays data for a staffmon's movepool, custom move, and custom ability."],``